src/js/misc.js

```diff
@@ -283,6 +285,8 @@ function handleDOMLoaded() {
                     }
 
                     if (cmd !== "") {
+                        cmd = cmd.replace(githubRef, "<a href='http://github.com/istio/istio/blob/master/$1'>$1</a>");
```

Comment:
woa. You really became a JS expert!.. 
Can I suggest pointing to the release branch instead of master? Ideally, there would be some variable that would help us point to the right code branch.